packages/adyen-integration/src/adyenv3/adyenv3-payment-strategy.ts

```diff
@@ -137,7 +137,8 @@ export default class Adyenv3PaymentStrategy implements PaymentStrategy {
         if (
             payment.methodId === 'klarna' ||
             payment.methodId === 'klarna_account' ||
-            payment.methodId === 'klarna_paynow'
+            payment.methodId === 'klarna_paynow' ||
+            payment.methodId === 'blik'
```

Comment:
Do we make those fix only for Blik? Should we trigger Adyen widget validation for other providers?

Reply:
I was testing it and didn't see any difference after adding submit() to other payment methods. Actually even for blik there was no difference in validation, I forgot to remove this line